lua/nvim_lsp/julials.lua

```diff
@@ -5,10 +5,18 @@ configs.julials = {
   default_config = {
     cmd = {
         "julia", "--project", "--startup-file=no", "--history-file=no", "-e", [[
-        using LanguageServer;
         using Pkg;
-        server = LanguageServer.LanguageServerInstance(stdin, stdout, false, dirname(Pkg.Types.Context().env.project_file));
-        server.runlinter = true; run(server);
+        Pkg.instantiate()
+        using LanguageServer; using SymbolServer;
+        depot_path = get(ENV, "JULIA_DEPOT_PATH", "")
+        project_path = dirname(Pkg.Types.Context().env.project_file)
```

Comment:
Is this the same as `Base.active_project()`?

Reply:
Ah, yes, I could use that instead then.

Reply:
Is that what you would suggest?

Reply:
If it works, yes. It seems better than digging so many layers into the Pkg internals.

Reply:
@kdheepak Mention me if you decide what to do here.

Reply:
@KristofferC, I made the change you suggested and it works. So I'll resolve this conversation.